docs/src/main/paradox/discovery/kubernetes.md

```diff
@@ -19,25 +19,43 @@ Using `akka-discovery-kubernetes-api` is very simple, as you simply need to depe
 
 And configure it to be used as default discovery implementation in your `application.conf`:
```

Comment:
This line needs to be updated as well, since it is no longer used as the default, only for cluster bootstrap.

Reply:
```suggestion
And configure it to be used for cluster bootstrap in your `application.conf`:
```